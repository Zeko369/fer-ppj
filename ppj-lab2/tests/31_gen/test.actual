err IDN 2 y
