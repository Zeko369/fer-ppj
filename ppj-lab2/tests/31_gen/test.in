IDN 1 n
OP_PRIDRUZI 1 =
OP_PLUS 1 +
IDN 1 n
OP_PUTA 1 *
L_ZAGRADA 1 (
OP_MINUS 1 -
IDN 1 n
D_ZAGRADA 1 )
OP_PUTA 1 *
BROJ 1 1201
IDN 2 y
OP_PRIDRUZI 2 =
BROJ 2 248
OP_PUTA 2 *
OP_MINUS 2 -
IDN 2 y
OP_PLUS 2 +
BROJ 2 1221
