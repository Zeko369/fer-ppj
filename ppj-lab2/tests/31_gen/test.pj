n = +n*(-n)*1201
y = 248*-y + 1221
