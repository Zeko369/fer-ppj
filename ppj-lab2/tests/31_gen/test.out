<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 n
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       OP_PLUS 1 +
       <P>
        IDN 1 n
      <T_lista>
       OP_PUTA 1 *
       <T>
        <P>
         L_ZAGRADA 1 (
         <E>
          <T>
           <P>
            OP_MINUS 1 -
            <P>
             IDN 1 n
           <T_lista>
            $
          <E_lista>
           $
         D_ZAGRADA 1 )
        <T_lista>
         OP_PUTA 1 *
         <T>
          <P>
           BROJ 1 1201
          <T_lista>
           $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 y
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 248
       <T_lista>
        OP_PUTA 2 *
        <T>
         <P>
          OP_MINUS 2 -
          <P>
           IDN 2 y
         <T_lista>
          $
      <E_lista>
       OP_PLUS 2 +
       <E>
        <T>
         <P>
          BROJ 2 1221
         <T_lista>
          $
        <E_lista>
         $
   <lista_naredbi>
    $
